giskard/scanner/report.py

```diff
@@ -18,7 +19,7 @@
 
 
 class ScanReport:
-    def __init__(self, issues, model=None, dataset=None, as_html: bool = True):
+    def __init__(self, issues, model=None, dataset=None, detectors_names=None, as_html: bool = True):
```

Comment:
i thought the idea was to remove this argument. Do we really need to know which detector didn't produce issues?